src/diagonal.works/b6/ingest/features_test.go

```diff
@@ -71,6 +73,38 @@ func TestCloneRelation(t *testing.T) {
 	}
 }
 
+func TestCloneCollection(t *testing.T) {
+	collection := CollectionFeature{
+		CollectionID: b6.MakeCollectionID(b6.NamespacePrivate, 1),
+		CollectionProto: pb.CollectionProto{
+			Keys: []*pb.LiteralNodeProto{
+				&pb.LiteralNodeProto{
+					Value: &pb.LiteralNodeProto_FeatureIDValue{
+						FeatureIDValue: &pb.FeatureIDProto{
+							Type:      pb.FeatureType_FeatureTypeCollection,
```

Comment:
i left this comment somewhere else too / not sure type should be collection here